pkg/controllers/cassandra/cluster_control.go

```diff
@@ -73,6 +73,12 @@ func NewControl(
 }
 
 func (e *defaultCassandraClusterControl) Sync(c *v1alpha1.CassandraCluster) error {
+	if c.Spec.Paused != nil && *c.Spec.Paused == true {
+		glog.V(4).Infof("defaultCassandraClusterControl.Sync skipped, since cluster is paused")
+		e.recorder.Eventf(c, apiv1.EventTypeNormal, "spec.paused", "Cluster paused, not syncing")
```

Comment:
The event name here (`spec.paused`) should be pulled into a const, and formatted similar to the other events.

Reply:
(we also need to move them into `pkg/apis/navigator/v1alpha1` at some point)